tests/utils/test_uri.py

```diff
@@ -27,6 +27,12 @@ def test_get_db_profile_from_uri_casing():
     assert get_db_profile_from_uri('databricks://aAbB') == 'aAbB'
 
 
+def test_get_db_path_info_from_profile():
+    assert get_db_path_info_from_uri('databricks://profile/prefix') == 'prefix'
+    assert get_db_path_info_from_uri('nondatabricks://profile/prefix') is None
+    assert get_db_path_info_from_uri('databricks://profile') is None
```

Comment:
try `'databricks://profile/'`, `'databricks://'`, `'databricks:/profile/prefix'`, `'databricks'`  ?

Reply:
Changed the test

One point of clarification: I'm pretty sure we currently don't support `databricks:/profile` because in `get_db_profile_from_uri`, if there aren't 2 slashes, then the netloc is empty, and that's the profile that we're supposed to read in.

Reply:
mm yeah.. i guess it'd be better to test that we throw something reasonable for that in `get_databricks_host_creds` (if we change it to take in the URI as the param)